### PR TITLE
User assign- Google things implemented

### PR DESCRIPTION
**Pull request by sdwowk**
HAD TO CHANGE THE ANDROID XML. please do not make changes to your branches until you have merged this!!

### CODE REVIEW

File: QuestionApp/AndroidManifest.xml [hunk 1]
```diff
@@ -7,6 +7,8 @@
     <uses-sdk
         android:minSdkVersion="17"
         android:targetSdkVersion="21" />
+    <uses-permission android:name="android.permission.ACCOUNT_MANAGER"/>
```
Comment (stephenjust) [inline QuestionApp/AndroidManifest.xml:10]: Apparently this only works on rooted phones. It's triggering a lint failure... It doesn't seem wrong though.

Comment (stephenjust) [inline QuestionApp/AndroidManifest.xml:10]: Confirmed. No working. Even on my real phone, the app can't see any accounts.

**Comment by stephenjust** [comment]
Your google account code is non-functional. It worked in an older API, but no longer works due to security stuff.
Either try [this](http://stackoverflow.com/questions/6502017/getting-android-owners-email-address-nicely/19444640#19444640) or remove the google account stuff.

**Comment by sdwowk** [comment]
I'm using a nexus 4 and it works.

**Comment by stephenjust** [comment]
Is it a rooted nexus 4? From what I read, the problem is that the
ACCOUNT_SERVICE permission is only for system apps (as of Android 4), and
it just silently fails anywhere else. Unless you have a rooted phone, which
let's everything be a system app.

**Comment by sdwowk** [comment]
I will try to fix it!

**Comment by stephenjust** [comment]
I merged your first commit that works. Please fix the google stuff and open a new pull-request with that fixed code.

**Pull request closed without merging**